omero/sysadmins/in-place-import.rst

```diff
@@ -4,7 +4,7 @@ In-place import
 ===============
 
 In-place import allows files which are already present on the server machine
-to be imported into OMERO without the need to copy them into OMERO's ManagedRepository. This requires
```

Comment:
This might be a question for @fmeyenhofer. He added this specifically trying to get something across.